### PR TITLE
Clearer clear and reset on step2

### PR DESCRIPTION
**Pull request by pmbohm**
This is all about making the reset link (some buttons) appear to be uniform throughout the site.
It is a foreign order from @pblain

### CODE REVIEW

File: web-app/js/portal/cart/DownloadPanel.js [hunk 1]
```diff
@@ -134,23 +134,25 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
     },
 
     initButtonPanel: function () {
+
+         this.resetLink = new Ext.ux.Hyperlink({
+            text: OpenLayers.i18n("clearLinkLabel", {text: 'Clear and Reset'}) ,
```
Comment (dnahodil) [inline web-app/js/portal/cart/DownloadPanel.js]: The 'Clear and Reset' text should be loaded with i18n as well.

(This happens a few times in this PR, I won't comment them all)

File: web-app/js/portal/details/NcWmsPanel.js [hunk 2]
```diff
@@ -46,17 +46,15 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
...
-                listeners: {
-                    scope: this,
-                    click: this.resetConstraints
-                }
-            })
-        );
+        this.resetLink = new Ext.ux.Hyperlink({
+            anchorCls: 'resetText',
+            cls: "clearFiltersLink",
+            text: OpenLayers.i18n('clearLinkLabel', {text: 'Clear and Reset'})
```
Comment (dnahodil) [inline web-app/js/portal/details/NcWmsPanel.js]: This is changing the text of the button from whatever is in `clearFilterButtonLabel` to 'Clear and Reset'. Is that deliberate or a mistake?

File: web-app/js/portal/filter/ui/FilterGroupPanel.js [hunk 3]
```diff
@@ -272,17 +271,5 @@ Portal.filter.ui.FilterGroupPanel = Ext.extend(Ext.Container, {
...
-    _isDisplayed: function() {
-        var foundHiddenParent = false;
-
-        this.bubble(function() {
-            if (this.isVisible && !this.isVisible()) {
-                foundHiddenParent = true;
-            }
-        });
-
-        return !foundHiddenParent;
```
Comment (dnahodil) [inline web-app/js/portal/filter/ui/FilterGroupPanel.js]: Nice work getting rid of unused code.

**Comment by dnahodil** [comment]
There's also the 'New search' button on Step 1. Should that come into line as well?

**Comment by dnahodil** [comment]
Comments made @pmbohm

**Comment by pmbohm** [comment]
@dnahodil Comments addressed. Mistakes fixed. Omissions added. thanks!
Step1 and step3 changes to come in a separate pull request

**Comment by dnahodil** [comment]
Looks good

**Pull request merged**